app/jobs/delete_old_blackouts_job.rb

```diff
@@ -0,0 +1,36 @@
+class DeleteOldBlackoutsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform
+    return if disabled
+    run
+    clean
+    Rails.logger.info 'Finished deleting old blackouts.'
+  end
+
+  private
+
+  def disabled
+    AppConfig.check(:blackout_exp_time, '').blank?
+  end
+
+  def run
+    old_blackouts.each do |b|
+      log_deletion b
+      b.destroy
+    end
+  end
+
+  def old_blackouts
+    threshold = Time.zone.today - AppConfig.get(:blackout_exp_time).days
```

Comment:
Not that it really matters, but is there a way to avoid running this line every time we call `old_blackouts`? I know that the performance impact is probably small, but just asking out of curiosity...